### ISSUE TITLE
Add record/column for days in inventory.   To identify vehicles that have been on lot for longer

### ISSUE THREAD

**Issue by Chrismac88** [Ben-Chapman/EVFinder#78]
**Is your feature request related to a problem? Please describe.**
The longer a vehicle has been on a dealer’s lot, the more negotiating power the buyer has.   Would have to import into a spreadsheet or database myself to start tracking this.

**Describe the solution you'd like**
If there was just a date field for when it was added to inventory, or a field that autocalculates how long it’s been in inventory

**Describe alternatives you've considered**
Importing the data into a spreadsheet

**Additional context**

**Comment by Ben-Chapman** [comment]
Hey @Chrismac88, thanks for the issue.

I try to present detailed availability data for the manufacturer's which provide this information. You can see this primarily with Hyundai, and somewhat with the Mustang Mach-E. Most manufacturers don't expose this data through their public APIs, so sometimes I'm left with "Available" or "Arriving Soon" (Kia). 

If you see more detailed availability information on a manufacturer's website, and not on the EV Finder please let me know and I'll take a look.

Long story short, I do what I can with the data I have - but that data most times isn't as granular as I (or you) would like :)
